docs/pages/versions/unversioned/sdk/securestore.mdx

```diff
@@ -107,6 +114,38 @@ When using `expo-secure-store`, you can set the [`ios.config.usesNonExemptEncryp
 
 Setting this property automatically handles the compliance information prompt.
 
+## Android Auto Backup
+
+[Android Auto Backup for Apps](https://developer.android.com/identity/data/autobackup) automatically backs up a user's data from apps that target and run on Android 6.0 (API level 23) or higher.
+The Auto Backup system has to be configured to exclude `expo-secure-store` shared preferences entries, as it's impossible to decrypt them after
+restoring the backup &mdash; app's entries are deleted from the Android Key Store when the app is uninstalled.
```

Comment:
```suggestion

The Auto Backup system has to be configured to exclude `expo-secure-store` shared preferences entries, as it's impossible to decrypt them after restoring the backup &mdash; app's entries are deleted from the Android Key Store when the app is uninstalled.
```

- Let's break this from above paragraph.
- These new changes look good, thank you for clarifying!